rocky/tests/test_crisis_room.py

```diff
@@ -31,13 +29,16 @@ def test_crisis_room_observed_at(rf, client_member, mock_crisis_room_octopoes):
     request.resolver_match = resolve(reverse("crisis_room"))
     response = CrisisRoomView.as_view()(request)
     assert response.status_code == 200
-    assertContains(response, "Jan 01, 2021")
```

Comment:
Removed this date before filter
<img width="404" alt="image" src="https://github.com/user-attachments/assets/35ebf640-1259-4ab0-a435-2235340374f6">

Reply:
Moved to top:
<img width="752" alt="image" src="https://github.com/user-attachments/assets/57832264-fbd2-4603-a781-29410dd973f0">